Evaluation/Input/Content/Section2.2_Data.md

```
@@ -0,0 +1,100 @@
+| [Kakuda 2014](#main-references)| CYP2D6 EM subjects with a single PO dose of 30 mg sd|
+| [Khalilieh 2018](#main-references)| CYP2D6 EM subjects with a single PO dose of 30 mg sd|
+| [Nakashima 2007](#main-references)| CYP2D6 EM subjects with a single PO dose of 50 mg sd|
+| [Nyunt 2008](#main-references)| CYP2D6 EM subjects with a single PO dose of 30 mg sd|
+| [Sager 2014](#main-references)| CYP2D6 EM subjects with a single PO dose of 30 mg sd|
+| [Stage 2018](#main-references)| CYP2D6 EM subjects with a single PO dose of 30 mg sd|
+
+#### DGI Model Building <a id="base model-building"></a>
+
+The following studies were used for model building (training data):
+
+**Publication** | **Study description**|
```

markdownlint-cli2 (0.21.0)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: trailing_only; Missing leading pipe

(MD055, table-pipe-style)